packages/core/README.md

```diff
@@ -62,13 +63,13 @@ sass)
 Using Yarn
 
 ```sh
-$ yarn add @carbon/vue
+$ yarn add @carbon/vue --dev
```

Comment:
Hey @rodet I do rather think of the component library as a full dependency.

If I read https://medium.com/@dylanavery720/npmmmm-1-dev-dependencies-dependencies-8931c2583b0c correctly, I would put it on a part with express and ajax, not babel and eslint.

Reply:
OK that's a valid point. I have had PR where developers asked me the opposite (ie dependencies are dependencies are included unprocessed as is in the deliverable, like Express on Node) - so I'll follow the project guidance.